balancer-js/src/modules/pools/factory/composable-stable/composable-stable.factory.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
```

Comment:
Lets remove these tests and instead test against a local fork checking for `PoolCreated` event.